docker/px4-dev/Dockerfile_ros-kinetic

```diff
@@ -20,6 +20,8 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-key 421C365BD9F
 		protobuf-compiler \
 		python-catkin-tools \
 		python-tk \
```

Comment:
Can these change as well?

Reply:
`python-catkin-tools` you can't.

Reply:
Actually, I don't even know why we need Tkinter here. Do you know why?

Reply:
The `python3-catkin-tools` package doesn't work?

Reply:
Tkinter might be needed for the ecl analysis script. https://github.com/PX4/Firmware/tree/master/Tools/ecl_ekf

Reply:
> The `python3-catkin-tools` package doesn't work?

Not saying it doesn't work, but this tool is used exclusively for ROS builds, so it makes sense to me that the tools used in ROS remain synced on the same Python version.